Dockerfiles/ci/README.md

```diff
@@ -0,0 +1,37 @@
+# CVP upstream image for testing in github actions
+CVP upstream image operator bundle validate container
+
+# Usage
+
+## Building image on local machine 
+
+```
+git clone https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
+cd operator-test-playbooks
+
+### podman build -t <image_name>:<tagname> -f <filepath> --build-arg OPERATOR_SDK_VERSION=<operator_sdk_version>
+### example:
+podman build -t upstream_ci_image:latest -f Dockerfiles/ci/Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0
+```
+
+## Running tests locally with built image
+
+```
+// create empty directories for testing
+mkdir test_operator_work_dir output_logs
+podman run -it -v $PWD:/project/operator-test-playbooks -v ./Dockerfiles/ci/example-metadata-without-alm-annotations:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=test_for_report_failed_empty_alm_examples -e OPERATOR_DIR=/project/operator_dir -e WORK_DIR=/project/output -e OPERATOR_WORK_DIR=/project/test_operator_work_dir/ upstream_ci_image:latest
+```
+```
+Note: currently there is only one test inside the container image i.e., test_for_report_failed_empty_alm_examples
+Even though the environment variable is not mentioned container tries to run all the tests by default. 
```

Comment:
I don't know what environment variable you are referring to. Could you flesh this out a bit?

Reply:
Outdated docs will update.